src/pages/LandingPage/LandingPage.jsx

```diff
@@ -25,9 +49,10 @@ const LandingPage = () => {
             </div>
             <div className="md:w-1/2">
               <img
+                onClick={() => handleClick()}
                 src={heroImage}
                 alt="Photography of Sahil"
-                className="w-3/4 h-auto rounded-full object-cover mx-auto"
+                className="w-3/4 h-auto rounded-full object-cover mx-auto cursor-pointer transition-scale duration-300 hover:outline"
```

Comment:
Good choices for the hover effect! However, i regretted the scale suggestion, it looked a bit silly, so i removed it.